HUD/SetBlipDisplay.md

```diff
@@ -9,19 +9,25 @@ void SET_BLIP_DISPLAY(Blip blip, int displayId);
 ```
 
 **displayId Behaviour** <br>
```

Comment:
You left a `<br>` here

Reply:
This br is here so that there is a line break between the title and the table.